### DISCUSSION TITLE
Updating to 2.1

### DISCUSSION THREAD

**Discussion by MasterOfMouse** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#140] [Q&A]
I'm on 2.0, do I just simply drag the 2.1 file into the Applications folder and click on 'Replace' in the pop-up box? Do I have to close the app before doing so?
<img width="472" alt="Screenshot 2021-02-24 at 5 28 02 PM" src="https://user-images.githubusercontent.com/79579320/108997558-bc04d400-76c5-11eb-961d-116aafcb69e8.png">
I apologise if this is a stupid question. I very recently got into the Mac scene with the M1.

**Comment by MatthiasKerbl** [comment]
> I'm on 2.0, do I just simply drag the 2.1 file into the Applications folder and click on 'Replace' in the pop-up box? Do I have to close the app before doing so?
> <img alt="Screenshot 2021-02-24 at 5 28 02 PM" width="472" src="https://user-images.githubusercontent.com/79579320/108997558-bc04d400-76c5-11eb-961d-116aafcb69e8.png">
> I apologise if this is a stupid question. I very recently got into the Mac scene with the M1.

Hi @MasterOfMouse,
no worries, I was confused about this too when I started using Macs.
I would recommend you the following steps to update AlDente:
1. Set Max. Battery Charge to 100%
2. Quit AlDente
3. Delete the old program file from the applications folder
4. Paste the new program file into the applications folder
5. Start the new AlDente

Best regards, Matthias

**Reply by MasterOfMouse** [reply]
Thank you @TheMazus for the reply! Much appreciated, I hope the Pro version launches soon!
